### DISCUSSION TITLE
Hide/reveal the menu

### DISCUSSION THREAD

**Discussion by loaonline** [pear-devs/pear-desktop#140] [Q&A] [answered]
Great app!  If I select to hide the menu bar in config, is there a quick way to reveal it?

**Comment by Araxeus** [comment, marked as answer]
Yes ! use `alt` key
or with `in-app-menu` plugin  **:**  `escape` key

**Comment by inson1** [comment]
@loaonline May you close it? :) You have got already your answer.

**Reply by loaonline** [reply]
yep!
